integration-tests/src/test/java/oracle/kubernetes/operator/MiiBaseTest.java

```diff
@@ -116,4 +116,57 @@ public void modifyDomainYamlWithNewConfigMapAndDomainRestartVersion(
     LoggerHelper.getLocal().log(Level.INFO, exec.stdout());
 
   }
+
+  /**
+   * Modify the domain yaml to change domain-level restart version.
+   * @param domain the domain
+   * @param versionNo version number of domain
+   *
+   * @throws Exception on failure
+   */
+  protected void modifyDomainYamlWithRestartVersion(
```

Comment:
In the method modifyDomainYamlWithRestartVersion can't we use  kubectl patch command to update the Restart Version instead of modifying the domain.yaml file. We can get the current version and add +1 to it.  No need to get VersionNumber parameter. It will be easier to port to Junit5 where we will be using kubectl API.

Reply:
solved